doc/subsystems/usb/usb.rst

```diff
@@ -1,5 +1,13 @@
+USB device stack
```

Comment:
For inter-document linking with ``:ref:`name` ``, we add a unique label to the top of all documents:

    .. _usb_device_stack:

    USB device stack
    ################